eos-updater-flatpak-installer/installer.h

```diff
@@ -73,13 +73,13 @@ typedef enum {
 
 gboolean eos_updater_flatpak_installer_check_ref_actions_applied (FlatpakInstallation  *installation,
                                                                   const gchar          *pending_flatpak_deployments_state_path,
-                                                                  GHashTable           *table,
+                                                                  GPtrArray            *actions,
```

Comment:
This commit (`flatpak: Flatten and compress all the actions after concatenating them`) looks good to be squashed into the main commit which introduces the autoinstall lists. I can’t see any issues with it. 👍

Reply:
Done